modules/integration/tests-common/integration-test-utils/src/main/java/org/wso2/identity/integration/common/utils/ISIntegrationTest.java

```diff
@@ -143,6 +148,62 @@ public void setSystemproperties() {
         System.setProperty("javax.net.ssl.trustStoreType", "JKS");
     }
 
+
+    /**
+     * Get the qualified endpoint URL with the hostname for the given tenant.
+     *
+     * @param endpointURL   The endpoint URL with the hostname.
+     * @param tenantDomain  Tenanted domain.
+     * @return Tenant qualified URL.
+     */
+    public String getTenantQualifiedURL(String endpointURL, String tenantDomain) {
+
+        try {
+            if(!tenantDomain.equalsIgnoreCase(MultitenantConstants.SUPER_TENANT_DOMAIN_NAME)) {
+                String baseURL = getBaseURL();
+                endpointURL = endpointURL.replace(baseURL, baseURL + TENANTED_URL_PATH_SPECIFIER + tenantDomain);
+            }
+            return endpointURL;
+        } catch (XPathExpressionException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    /**
+     * Get the qualified endpoint URL without the hostname for the given tenant.
+     *
+     * @param endpointURLWithHostname   The endpoint URL without the hostname.
+     * @param tenantDomain              Tenanted domain.
+     * @return Tenant qualified URL without hostname.
+     */
+    public String getTenantQualifiedURLWithoutHostName(String endpointURLWithHostname, String tenantDomain) {
+
+        if(!tenantDomain.equalsIgnoreCase(MultitenantConstants.SUPER_TENANT_DOMAIN_NAME)) {
+            endpointURLWithHostname = TENANTED_URL_PATH_SPECIFIER + tenantDomain + endpointURLWithHostname;
```

Comment:
don't we need to add "/"

Reply:
The `TENANTED_URL_PATH_SPECIFIER` contains the "/" before the tenant domain and `endpointURLWithHostname` always starts with a "/".